lib/gollum/public/gollum/javascript/gollum.dialog.js

```diff
@@ -101,7 +101,7 @@
 
       var id = fieldAttributes.id || 'upload';
       var name = fieldAttributes.name || 'file';
-      var action = fieldAttributes.action || '/uploadFile';
+      var action = fieldAttributes.action || '/upload_file';
```

Comment:
You could consider moving this file to `gollum.dialog.js` and using the path helper here, too

Reply:
Correction: move it to `gollum.dialog.js.erb`

Reply:
Agreed. I'm planning to restructure the uploading code base, though, so I'll do that in another PR if you don't mind.